kernel-install/50-mkosi.install

```diff
@@ -80,45 +59,17 @@ def main() -> None:
         allow_abbrev=False,
         usage="50-mkosi.install COMMAND KERNEL_VERSION ENTRY_DIR KERNEL_IMAGE INITRD…",
     )
-
-    parser.add_argument(
-        "command",
-        metavar="COMMAND",
-        help="The action to perform. Only 'add' is supported.",
-    )
-    parser.add_argument(
-        "kernel_version",
-        metavar="KERNEL_VERSION",
-        help="Kernel version string",
-    )
-    parser.add_argument(
-        "entry_dir",
-        metavar="ENTRY_DIR",
-        type=Path,
-        nargs="?",
-        help="Type#1 entry directory (ignored)",
-    )
-    parser.add_argument(
-        "kernel_image",
-        metavar="KERNEL_IMAGE",
-        type=Path,
-        nargs="?",
-        help="Kernel image",
-    )
     parser.add_argument(
         "initrds",
         metavar="INITRD…",
         type=Path,
         nargs="*",
         help="Initrd files",
     )
```

Comment:
This can be moved into shared code as well, the command line interface used by plugins is identical across plugins

Reply:
this is specific for this one, the other doesn't accept this parameter

Reply:
@bluca It doesn't work like that, the plugins don't define the command line interface, kernel-install does. And kernel-install will pass all the initrds it receives as command line arguments to all plugins. So each plugin has to be able to deal with that.

`mkosi-addon` doesn't have to do anything with the passed initrds, it can just ignore them, but the parser should parse them all the same.